packages/affiliate/src/Affiliate.php

```diff
@@ -1,25 +1,33 @@
 <?php
+namespace Automattic\Jetpack\Partners;
 
-if ( ! defined( 'ABSPATH' ) ) {
-	// Exit if accessed directly
-	exit;
+use Jetpack;
```

Comment:
As far as I understand, a package shouldn't do this - packages are expected to be independently distributable packages that don't need Jetpack to work.

This also signifies that there's additional functionality that might make sense to be ported to this package, or to a separate one.

Reply:
Thanks. Replaced using the Status package.